client/src/store/configureStore.tsx

```diff
@@ -8,7 +8,7 @@ import rootReducer from '../slices/index'
 export const persistConfig = {
   key: 'redux-store-root',
   storage,
-  whitelist: ['preferences', 'characters', 'onboarding', 'credentials', 'connection'],
+  whitelist: ['preferences', 'configuration', 'characters', 'onboarding', 'credentials', 'connection'],
```

Comment:
Configuration slices has been removed right?

```suggestion
  whitelist: ['preferences', 'characters', 'onboarding', 'credentials', 'connection'],
```